client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -85,16 +95,35 @@ class DesignTypeWithStoreStep extends Component {
 	};
 
 	renderChoice = ( choice ) => {
-		return (
-			<Card className="design-type-with-store__choice" key={ choice.type }>
-				<a className="design-type-with-store__choice-link"
-					href="#"
-					onClick={ this.handleChoiceClick( choice.type ) }>
-					{ choice.image }
-					<h2>{ choice.label }</h2>
-				</a>
-			</Card>
-		);
+		switch ( abtest( 'signupCopy' ) ) {
+			case 'newSignupCopy':
+				return (
```

Comment:
If the text is empty, the element (`p`) should collapse on itself and not be visible. Of course, we shouldn't be adding empty elements (for accessibility reasons, for example), but for the purposes of a test it seems OK.
You can also inject some logic into the markup, like this:
```
<p>{ abtest( 'something' ) === 'variation' ? 'New copy' : 'Old copy' }</p>
```
Or even pass around whole nodes:
```
let content = <p>{ this.translate( 'Old copy' ) }</p>;
if ( abtest ) {
   content = null;
}

return (
   <div>
      { content }
   </div>
);
```

`null` and `false` values are not rendered at all by React, so you can also use the above if it fits the use case better. The [official docs](https://facebook.github.io/react/docs/jsx-in-depth.html#javascript-expressions) go into more detail :)